tests/functional/spec/components/radio.spec.js

```diff
@@ -0,0 +1,124 @@
+const helpers = require('../../helpers');
+const RadioMandatoryPage = require('../../pages/surveys/radio/radio-mandatory.page');
+const RadioNonMandatoryPage = require('../../pages/surveys/radio/radio-non-mandatory.page');
+const SummaryPage = require('../../pages/surveys/radio/summary.page');
+
+describe('Component: Radio', function() {
+
+var radio_schema = 'test_radio.json';
+
+
+    describe('Optional', function() {
+          beforeEach('load the survey', function() {
+            return helpers.openQuestionnaire(radio_schema).then(() => {
+                return browser
+                  .click(RadioMandatoryPage.none())
+                  .click(RadioMandatoryPage.submit());
+                  });
+                });
+
+          it('When I submit without choosing an option that I should be able to go on to the next page', function() {
+             return browser
+               .click(RadioNonMandatoryPage.submit())
+               .getUrl().should.eventually.contain(SummaryPage.pageName);
+            });
+
+         it('When I select an optional option field and not enter any text I should still be able to move on', function() {
+           return browser
+              .getUrl().should.eventually.contain(RadioNonMandatoryPage.pageName)
+              .click(RadioNonMandatoryPage.other())
+              .click(RadioNonMandatoryPage.submit())
+
+              .getText(SummaryPage.radioMandatoryAnswer()).should.eventually.contain('None')
+              .getText(SummaryPage.radioNonMandatoryAnswer()).should.eventually.contain('No answer provided');
+
+            });
+
+        it('When I select the other option then submit then return and remove other field text then it should be displayed correctly on the summary page', function() {
+         return browser
+          .click(RadioNonMandatoryPage.other())
+          .setValue(RadioNonMandatoryPage.otherText(), 'Hello')
+          .click(RadioNonMandatoryPage.submit())
+
+          .click(SummaryPage.previous())
+
+          .click(RadioNonMandatoryPage.other())
+          .setValue(RadioNonMandatoryPage.otherText(), '')
+          .click(RadioNonMandatoryPage.submit())
+
+          .getText(SummaryPage.radioMandatoryAnswer()).should.eventually.contain('None')
+          .getText(SummaryPage.radioNonMandatoryAnswer()).should.eventually.contain('No answer provided');
+          });
+    });
+
+    describe('Mandatory', function() {
+      beforeEach('load the survey', function() {
+        return helpers.openQuestionnaire(radio_schema).then(() => {
+            return browser;
```

Comment:
i think this could just be `return helpers.openQuestionnaire(radio_schema)`
you shouldn't need the `then`?